CHANGELOG.md

```diff
@@ -1,16 +1,223 @@
+v1.5.0
+------
+* **Release date:** 2018-09-??.
+* **Requires:** [tileserver v2.2.0](https://github.com/mapzen/tileserver/releases/tag/v2.2.0) and [tilequeue v2.1.0](https://github.com/mapzen/tilequeue/releases/tag/v2.1.0)  and [rawr-tiles v1.0.0](https://github.com/mapzen/raw-tiles/releases/tag/v1.0.0) and [coanacatl v1.0.0](https://github.com/mapzen/coanacatl/releases/tag/v1.0.0).
+
+  #### ENHANCEMENTS
+
+  * **buildings**: add `entrance` points, with optional **kind_detail** property with values: `main`, `staircase`, `service`, `home`, `unisex`, `garage`, `residence`, `private`, or `secondary`.
```

Comment:
Do we usually sort lists of `kind_detail` alphabetically?

Reply:
Fixed in 4f182ab8da4b13c016cd38fc9db43ad749aad09b (and also in docs).